core/src/test/java/com/google/instrumentation/trace/LinkTest.java

```diff
@@ -18,26 +18,17 @@
 import com.google.common.testing.EqualsTester;
 import com.google.instrumentation.trace.Link.Type;
 import java.util.Random;
-import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 /** Unit tests for {@link Link}. */
 @RunWith(JUnit4.class)
 public class LinkTest {
-  private Random random;
-  private SpanContext spanContext;
-
-  @Before
-  public void setUp() {
-    random = new Random(1234);
-    spanContext =
-        new SpanContext(
-            TraceId.generateRandomId(random),
-            SpanId.generateRandomId(random),
-            TraceOptions.DEFAULT);
-  }
+  private static final Random random = new Random(1234);
+  private static final SpanContext spanContext =
+      SpanContext.create(
+          TraceId.generateRandomId(random), SpanId.generateRandomId(random), TraceOptions.DEFAULT);
```

Comment:
What is the reason for this change?  I thought it was better for the two fields to be non-static, because it made it clear that the tests are deterministic.  Even though `random` is only used to initialize `spanContext`, it looks like it could be used by individual tests, which could make the test results depend on the order in which they are run.  You could just make the fields non-static to ensure that they are initialized for each test.